### PR TITLE
Portfolio: setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by X-Elie-X**
## In this PR I implemented the below:
-Added linters files.
-Installed llinters.
-Created`index.html`
-Created`style.CSS` file to add some styling.
-Created `navbar` and `main section`.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  <body>
+    <header class="Header">
+      <nav>
+        <h1>Elie Harfouche</h1>
+        <div class="lines">
+          <div class="line1"></div>
+          <div class="line2"></div>
+          <div class="line3"></div>
+        </div>
+      </nav>
```
Comment (cynthiainga) [inline index.html:19]: - [ ] Good job so far but I think you need to change something at this section, your `<h1>` is a logo and the logo is supposed to be in the `<a>` tag, kindly fix it :+1: :+1:.
- [ ] For the hamburger button you suppose to use the image not the class that you style, and also it suppose to be in the `<nav>` tag not the logo :upside_down_face: :upside_down_face:
Example of how you can implement it:
```
  <a class="logo" href="#">Elie Harfouche</a>
  <nav class="nav-bar">
     <a href="#">
         <img src="image-link">
     </a>
  </nav>
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    </header>
+    <main>
+      <p class="Geading">Hey There. I’m Elie</p>
```
Comment (cynthiainga) [inline index.html:22]: - [ ] Please kindly change the `<p>` tag to `<h1>`, is not the paragraph but the title :+1: :+1:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      <p class="dev">I’am a Software Developer</p>
+      <p class="info">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
```
Comment (cynthiainga) [inline index.html:28]: - [ ] Good implementation of the paragraph but I think you can put all in one `<p>` tag, so to go to the next line you can use the `<br/>` tag :upside_down_face:

File: index.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      <div class="sm">
+        <a href="#"><img src="./Vector.png " alt=" " /></a>
+        <a href="#"><img src="./Group.svg" alt="" /></a>
+        <a href="#"><img src="./Vector1.svg" alt="" /></a>
+        <a href="#"><img src="./Vector2.svg " alt="" /></a>
+        <a href="#"><img src="./Vector.svg" alt="" /></a>
```
Comment (cynthiainga) [inline index.html:34]: - [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag) :upside_down_face:

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @X-Elie-X , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there :pinching_hand::pinching_hand:!



## Highlights
- [x] No linters errors :heavy_check_mark: 
- [ ] Good commit history :heavy_check_mark: 
- [ ] Good description of your PR :heavy_check_mark: 


## Required Changes ♻️

- [ ] Please kindly check the comments under the review for more details :point_down: :point_down: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @X-Elie-X       , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

[OPTIONAL] Kindly add a screenshot to your README file showing your webpage user Interface. You can use extentions like fireshot to capturer the entire page.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**